src/app/app.component.html

```diff
@@ -1,9 +1,7 @@
 
-<div fxLayout="row" class="app-container">
+<div  fxLayout="row" class="app-container">
```

Comment:
please remove extra space here

Reply:
I have resolved it la......................Thank you